protocol/test/foundry/PipelineConvert.t.sol

```diff
@@ -0,0 +1,1461 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0 <0.9.0;
+pragma abicoder v2;
+
+import {TestHelper} from "./utils/TestHelper.sol";
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import {LibTransfer} from "contracts/libraries/Token/LibTransfer.sol";
+import {IMockFBeanstalk} from "contracts/interfaces/IMockFBeanstalk.sol";
+import {MockSeasonFacet} from "contracts/mocks/mockFacets/MockSeasonFacet.sol";
+import {MockSiloFacet} from "contracts/mocks/mockFacets/MockSiloFacet.sol";
+import {MockPump} from "contracts/mocks/well/MockPump.sol";
+import {ConvertFacet} from "contracts/beanstalk/silo/ConvertFacet.sol";
+import {Bean} from "contracts/tokens/Bean.sol";
+import {IWell, Call} from "contracts/interfaces/basin/IWell.sol";
+import {FarmFacet} from "contracts/beanstalk/farm/FarmFacet.sol";
+import {SeasonGettersFacet} from "contracts/beanstalk/sun/SeasonFacet/SeasonGettersFacet.sol";
+import {ConvertGettersFacet} from "contracts/beanstalk/silo/ConvertGettersFacet.sol";
+import {MockToken} from "contracts/mocks/MockToken.sol";
+import {Pipeline} from "contracts/pipeline/Pipeline.sol";
+import {DepotFacet, AdvancedPipeCall} from "contracts/beanstalk/farm/DepotFacet.sol";
+import {AdvancedFarmCall} from "contracts/libraries/LibFarm.sol";
+import {SiloGettersFacet} from "contracts/beanstalk/silo/SiloFacet/SiloGettersFacet.sol";
+import {C} from "contracts/C.sol";
+import {console} from "forge-std/console.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {LibWell} from "contracts/libraries/Well/LibWell.sol";
+import {ICappedReservesPump} from "contracts/interfaces/basin/pumps/ICappedReservesPump.sol";
+import {LibClipboard} from "contracts/libraries/LibClipboard.sol";
+import {LibWellMinting} from "contracts/libraries/Minting/LibWellMinting.sol";
+import "forge-std/Test.sol";
+
+contract MiscHelperContract {
+    function returnNumber(uint256 num) public pure returns (uint256) {
+        return num;
+    }
+}
+
+/**
+ * @title PipelineConvertTest
+ * @author pizzaman1337
+ * @notice Test pipeline convert.
+ */
+contract PipelineConvertTest is TestHelper {
+    using SafeMath for uint256;
+
+    // Interfaces.
+    // IMockFBeanstalk bs = IMockFBeanstalk(BEANSTALK);
+    MockSiloFacet silo = MockSiloFacet(BEANSTALK);
```

Comment:
The purpose of `IMockFBeanstalk bs = IMockFBeanstalk(BEANSTALK);` was to provide a general interface for functions that aren't directly being tested. This massively cleans up the instantaniation of various variables. Imo, this test should have a similar amount of variables being initialized as `convert.t.sol`.